lib/plugins/globalid.js

```diff
@@ -8,17 +8,27 @@ var invariant = require('../utils/misc').invariant
 var noop = require('../utils/misc').noop
 var cookie = require('../vendor/js-cookies')
 
-function cacheSuccess (result, cookieName) {
+function cacheSuccess (result, cookieName, cookieOptions) {
+  cookieOptions = cookieOptions || {}
+
   if (!result['global_id']) {
     return null
   }
-  cookie.setItem(cookieName, result['global_id'], 6000)
+  var path = cookieOptions.path || ''
+  var domain = cookieOptions.domain || ''
+  var secure = cookieOptions.secure || ''
+  var maxAge = cookieOptions.maxAge || 6000
```

Comment:
Not sure why we need to provide defaults for these value? The current logic should work fine with undefined.

Reply:
It is just a habit for setting default values. I think it is ok to leave it as is